pkg/machine/vmconfigs/machine.go

```diff
@@ -320,6 +320,13 @@ func (mc *MachineConfig) ConnectionInfo(vmtype define.VMType) (*define.VMFile, *
 	return socket, getPipe(mc.Name), err
 }
 
+func (mc *MachineConfig) GetAddress() string {
+	if mc.HyperVHypervisor != nil && mc.HyperVHypervisor.IPAddress != "" {
+		return mc.HyperVHypervisor.IPAddress
```

Comment:
Before this PR, compilation is broken on non-windows platforms:
```
pkg/machine/ssh.go:41:33: mc.HyperVHypervisor.IPAddress undefined (type *vmconfigs.HyperVConfig has no field or method IPAddress)
```
I think this PR will have the same issue. Might be best to move `IPAddress` to `MachineConfig` directly, there are already hypervisor-specific fields there (`Rosetta`)